cmd/helm/release_testing.go

```diff
@@ -18,28 +18,39 @@ package main
 
 import (
 	"io"
+	"time"
 
 	"github.com/spf13/cobra"
 
+	"helm.sh/helm/cmd/helm/require"
```

Comment:
Helm moved to Go modules. That means the major version needs to be in the import path. This applies when something has a major version of 2 or higher. So, the helm import paths need to have a /v3 in them. For example,

`"helm.sh/helm/cmd/helm/require"` -> `"helm.sh/helm/v3/cmd/helm/require"`